src/sentry/runner/commands/devservices.py

```diff
@@ -48,11 +50,64 @@ def devservices():
     """
 
 
+@devservices.command()
+@click.option("--project", default="sentry")
+@click.argument("service", nargs=1)
+def attach(project, service):
+    """
+    Run a single devservice in foreground, as opposed to `up` which runs all of
+    them in the background.
+
+    Accepts a single argument, the name of the service to spawn. The service
+    will run with output printed to your terminal, and the ability to kill it
+    with ^C. This is used in devserver.
+
+    Note: This does not update images, you will have to use `devservices up`
+    for that.
+    """
+
+    os.environ["SENTRY_SKIP_BACKEND_VALIDATION"] = "1"
+
+    from sentry.runner import configure
+
+    configure()
+
+    client = get_docker_client()
+    containers = _prepare_containers(project)
+    container = _start_service(client, service, containers, project, fast=True)
+
+    def exit_handler(*_):
```

Comment:
Why `*_`? Fairly sure that's not needed.

Reply:
signal handlers take `signum, frame` while atexit takes nothing

Reply:
Oh sorry, overlooked the fact you were reusing the same handler for both atexit and signals. For context, I'm used to doing something like:

```
def sig_exit_handler(signo, frame):
    exit_handler()
    sys.exit(128 + signo)  # http://tldp.org/LDP/abs/html/exitcodes.html
```